app/src/main/java/com/jerboa/ui/components/post/composables/PostComposables.kt

```diff
@@ -178,6 +180,22 @@ fun CreateEditPostBody(
                 keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Uri),
                 modifier = Modifier.fillMaxWidth(),
             )
+
+            /**
+             * Pick and upload an image, show a preview if possible
+             */
+            PickImage(
```

Comment:
I see that we use this in a lot of places but does this actually work on API level 26-28 ?

Reply:
Not sure, but I don't see any API warnings for versions. Somebody would have to open an issue if there are.